specs/latest/2.0/index.html

```diff
@@ -415,6 +415,7 @@ <h2 class="no-toc">Table of contents</h2>
 typedef (Int32Array or sequence&lt;GLint&gt;) ClearBufferIVSource;
 typedef (Uint32Array or sequence&lt;GLuint&gt;) ClearBufferUIVSource;
 typedef (Float32Array or sequence&lt;GLfloat&gt;) ClearBufferFVSource;
+typedef (ArrayBuffer or SharedArrayBuffer) ByteBuffer;
```

Comment:
It's simpler to just accept (Shared)ArrayBuffer where alignment doesn't matter, since it's absolutely trivial to pull out the ArrayBuffer from a view.
I definitely prefer this for all new entrypoints, but I'll cede that existing WebGL1 entrypoints should probably retain view support for backwards compat.